### DISCUSSION TITLE
Non-US location support for BigQuery

### DISCUSSION THREAD

**Discussion by archiewood** [evidence-dev/evidence#2395] [Ideas]
Currently you cannot set a region for your BigQuery database, so only the default region (US) is usable

**Comment by archiewood** [comment]
See an implementation in https://github.com/evidence-dev/sqltools-bigquery-driver/blob/264f413ea75450f6217d54567c0bb1c4f75cff70/src/ls/driver.ts#L44

**Comment by plotti** [comment]
whats the status here? should i patch this myself or ...

**Comment by hughess** [comment]
@plotti we don't have this in progress yet. If you're willing to contribute this, that would be awesome!
